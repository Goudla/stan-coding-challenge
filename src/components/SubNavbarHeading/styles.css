.sub-navbar-heading {
  color: #fff;
  font-size: 26px;
  font-weight: 600;
  margin: 0;
  text-shadow: 0 1px 3px rgba(0,0,0,0.3);
}
