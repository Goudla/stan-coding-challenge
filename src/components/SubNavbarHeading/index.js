import * as React from 'react';

// Styles
import './styles.css';

type Props = {
  children: React.Node
};

class SubNavbarHeading extends React.PureComponent<Props> {
  render() {
    const { children } = this.props;

    return (
      <h2 className="sub-navbar-heading">
        {children}
      </h2>
    );
  }
}

export default SubNavbarHeading;
