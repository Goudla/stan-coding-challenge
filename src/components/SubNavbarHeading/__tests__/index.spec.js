import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

// Scenes
import SubNavbarHeading from '../';

describe('<SubNavbarHeading />', () => {
  it('Should render', () => {
    const renderer = new ShallowRenderer();
    renderer.render(<SubNavbarHeading>Test</SubNavbarHeading>);
    const result = renderer.getRenderOutput();
    expect(result).toMatchSnapshot();
  });
});
