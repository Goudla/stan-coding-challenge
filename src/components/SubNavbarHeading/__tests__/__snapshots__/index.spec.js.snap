// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SubNavbarHeading /> Should render 1`] = `
<h2
  className="sub-navbar-heading"
>
  Test
</h2>
`;
